src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,189 @@
+    }
+  } catch (exception: any) {
+    isError.value = true
+    errorLoadingConfiguration.value.notificationTitle = t(
+      'error.cannot_retrieve_netify_informatics_configuration'
+    )
+    errorLoadingConfiguration.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    loading.value = false
+  }
+}
+
+function clearErrors() {
+  error.value = {
+    notificationTitle: '',
+    notificationDescription: '',
+    uuid: ''
+  }
+}
+
+function handleSaveAutomaticToggle() {
+  clearErrors()
+  saving.value = true
+  const status = form.value.status ? 'enable' : 'disable'
+  ubusCall('ns.netifyd', status)
+    .then((response) => {
+      if (response.data && response.data.result && response.data.result === 'success') {
+        setTimeout(() => {
+          notificationsStore.createNotification({
+            title: t('standalone.netify_informatics.netify_informatics_' + status),
+            description: t(
+              'standalone.netify_informatics.netify_informatics_' + status + '_description'
+            ),
+            kind: 'success'
+          })
+        }, 500)
+      }
+    })
+    .catch((exception: AxiosError) => {
+      errorSaving.value.notificationTitle = t('error.cannot_save_configuration')
+      errorSaving.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    })
+    .finally(() => (saving.value = false))
+}
+
+function openDashboard() {
+  window.open('https://portal.netify.ai/login', '_blank')
+}
+function copyUuid() {
+  navigator.clipboard.writeText(form.value.uuid)
+  justCopied.value = true
+  setTimeout(() => {
+    justCopied.value = false
+  }, 3000)
+}
+</script>
+
+<template>
+  <div>
+    <FormLayout class="max-w-6xl">
+      <template #description>
+        <p>{{ t('standalone.netify_informatics.content_description') }}</p>
+        <NeButton kind="tertiary" size="lg" @click="openDashboard()" class="ml-2 shrink-0">
+          <template #prefix>
+            <FontAwesomeIcon :icon="faArrowUpRightFromSquare" aria-hidden="true" />
+          </template>
+          {{ t('standalone.netify_informatics.open_netify_dashboard') }}</NeButton
+        >
+      </template>
+      <NeSkeleton v-if="loading" :lines="5" />
+      <NeInlineNotification
+        v-if="errorLoadingConfiguration.notificationTitle"
+        class="my-4"
+        kind="error"
+        :title="errorLoadingConfiguration.notificationTitle"
+        :description="errorLoadingConfiguration.notificationDescription"
+      />
+      <div v-if="!isError && !loading" class="mb-8 flex flex-col gap-y-6">
+        <div class="relative flex items-center gap-2">
+          <NeTextInput
+            v-model="form.uuid"
+            disabled
+            :invalid-message="error.uuid"
+            :label="t('standalone.netify_informatics.uuid')"
+            ref="uuidRef"
+          >
+          </NeTextInput>
+          <NeButton kind="tertiary" size="sm" @click="copyUuid" class="ml-2.5 mt-8">
```

Comment:
"Copied to clipboard" badge should be a tooltip instead. A bit of rework is needed, let's replace this NeButton and the NeBadge with:

```suggestion
          <NeTooltip v-if="justCopied" triggerEvent="mouseenter focus" placement="top-start">
            <template #trigger>
              <NeButton kind="secondary" size="md" @click="copyUuid" class="mb-px ml-2.5">
                <template #prefix>
                  <FontAwesomeIcon :icon="faCopy" class="h-4 w-4" aria-hidden="true" />
                </template>
                {{ t('standalone.netify_informatics.copy') }}
              </NeButton>
            </template>
            <template #content>
              {{ t('standalone.netify_informatics.copied_to_clipboard') }}
            </template>
          </NeTooltip>
          <NeButton v-else kind="secondary" size="md" @click="copyUuid" class="mb-px ml-2.5">
            <template #prefix>
              <FontAwesomeIcon :icon="faCopy" class="h-4 w-4" aria-hidden="true" />
            </template>
            {{ t('standalone.netify_informatics.copy') }}
          </NeButton>
```